.github/workflows/windows.yml

```diff
@@ -0,0 +1,75 @@
+name: Windows CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+## events but only for the master branch
+on:
+  push:
+    branches:
+      - 'develop'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  tests:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: cache
+        id: cache-target
+        uses: actions/cache@v2
+        with:
+          path: c:/users/runneradmin/appdata/local/pip/cache
+          key: ${{ runner.os }}-${{ matrix.python-version }}
+
+      - name: Set up environment
+        run: |
+          # Update pip.
+          python -m pip install -U pip setuptools wheel
+
+          # Install dependencies for tests.
+          pip install --progress-bar=off -U -r tests/requirements-tools.txt -r tests/requirements-libraries.txt
+
+          # Install PyInstaller Hook Sample, to ensure that tests declared in
+          # entry-points are discovered.
+          pip install https://github.com/pyinstaller/hooksample/archive/v4.0rc1.zip
+
+          # Compile bootloader
+          cd bootloader
+          python waf distclean all
+          cd ..
+
+          # Install PyInstaller.
+          pip install --progress-bar=off -e .
+
+          # Make sure the help options print.
+          python -m pyinstaller -h
+
+      - name: Run tests
+        run: |
+          pytest -n 3 --maxfail 3 --durations 10 tests/unit tests/functional `
+            --force-flaky --no-flaky-report --junitxml=junit-results.xml
+
+          # The ``run_tests`` script is invoked with the ``-c`` option to
+          # specify a ``pytest.ini``, rather than allowing pytest to find
+          # something unexpected in the filesystem (it searches from the root
+          # dir all the way to the top of the filesystem per
+          # https://docs.pytest.org/en/latest/customize.html).
+          python -m PyInstaller.utils.run_tests -c PyInstaller/utils/pytest.ini --include_only=pyi_hooksample.
+
+      - name: Publish Test Report
+        uses: mikepenz/action-junit-report@v2
+        with:
+          report_paths: 'junit-results.xml'
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          fail_on_failure: true
```

Comment:
I'm the one who put it in.

Reply:
It shows the tests on appveyor (and with this, GitHub).

Reply:
I can see them anyway though. What's the difference? It appears to work identically on the [hooks repo](https://github.com/pyinstaller/pyinstaller-hooks-contrib/) which doesn't have junit.

Reply:
There's a message about insufficient permissions in the [workflow view](https://github.com/pyinstaller/pyinstaller/actions/runs/720243844).

Reply:
The permissions are probably insufficient because this is a fork. I suppose that means it can't run on pull requests because they will most likely be forks too?